packages/data-broker/src/services/dhis/DhisInputSchemeResolvingApiProxy.js

```diff
@@ -229,10 +229,7 @@ export class DhisInputSchemeResolvingApiProxy {
     for (let i = 0; i < dhisIds.length; i += SQL_BIND_CHUNK_SIZE) {
       const chunkDhisIds = dhisIds.slice(i, i + SQL_BIND_CHUNK_SIZE);
       const chunkMappings = await this.models.dataServiceEntity.find({
-        [QUERY_CONJUNCTIONS.RAW]: {
-          sql: `config->>'dhis_id' in (${chunkDhisIds.map(() => '?')})`,
-          parameters: chunkDhisIds,
-        },
+        'config->>dhis_id': chunkDhisIds,
```

Comment:
I'm fixing the unit tests I broke after changing this, but @IgorNadj you may be interested in this change. I've basically made it possible to reference jsonb field via a similar `->>` notation to the postgres one